### DISCUSSION TITLE
記事を改善する方法について

### DISCUSSION THREAD

**Discussion by daikisuyama** [increments/qiita-discussions#14] [要望・アイデア・不具合]
## 現状の課題

編集リクエストやコメントによる記事の改善が十分に機能していないと思われます。編集リクエストの場合、執筆者の意に沿った書き方ができるとは限らず、時間も要します。また、コメントの場合、公開されている所で指摘することに抵抗がある人も多いのではないでしょうか。

## 事例

改善が機能していないと感じた事例としては、以下の二つです。

- 自身の記事に明確な間違いがあっても、編集リクエストやコメントによる指摘を受けずに自分で数日後に直す場合がほとんど。
- Google検索の上位にある古い記事の内容が間違っていても誰も指摘しておらず、間違った記事が放置されている。

## 希望する機能

記事の修正希望を編集リクエストと似た形式で送信できる機能が欲しいです。また、修正希望の概要をタグで選択し、内容をコメントで説明する形式が良いのではないでしょうか。

**Comment by kskwtnk** [comment]
@daikisuyama 

ご意見ありがとうございます！

編集リクエスト機能そのものの認知度が低めであったりして、仰る通りうまく機能しきっていないと運営も認識しています。

https://qiita.com/about

aboutにも記載しているように、我々としてはお互いが手を取り合って良い情報共有が出来るのが理想であり、編集リクエストが上手く機能するサイクルを作っていきたいと思っています。

daikisuyama さんからいただいたままの機能になるかは確約できず申し訳ないのですが、目指す姿は一緒であると認識しています！
改善して、使いやすくなったと報告できるように頑張ります！

---

もし他の方でも編集リクエストやコメントについてご意見があれば、是非お願いします！
実は我々運営としてもかなり迷っているポイントであります……😭

**Comment by kokosabu** [comment]
編集リクエストは面白い機能だと思います。
タイポ見つけた時にコメントでわざわざ指摘するまででもないけど、直った方が良いよなと思うものを連絡するのに何度か使ったことがあります。
たまに、編集リクエストを送っても、採用も不採用もされずそのままっていうことがあったので、編集リクエストがあったことを通知する部分に改善する余地はあるのではないかと思っています。

**Reply by kyntk** [reply]
@kokosabu

> 編集リクエストを送っても、採用も不採用もされずそのままっていうことがあった

ありがとうございます。
運営もここは課題に思っておりまして、少し先にはなってしまうのですが、編集リクエストに気づいて機能する仕組みについて改善していきたいと思っています!

**Comment by daddygongon** [comment]
編集リクエストについての便乗です．

    編集リクエストを「送信」した後も編集したい

途中まで書いて，その後追加で編集リクエストを出すかどうか分からないときに，「下書き保存」ではなく，「送信」しています．

その後，送信した箇所を修正しようとしてもできなくて途方に暮れます．自分の記事ですと何度も修正しますが，編集リクエストは書いたことさえよく忘れるぐらいなので，「送信」を選択するのは間違いではないと思うのですが，編集リクエストの内容を編集リクエストしたいだけなのに．．．

**Reply by kskwtnk** [reply]
@daddygongon 

ありがとうございます！

こちら私も経験あります😔
恥ずかしい感じの誤字をしていたときとか、編集できずに少し焦りました……笑

ただ申し訳ありません、直近では送った編集リクエストを再編集しやすくするような機能開発は行えなさそうです
（どちらかと言うと、まずは編集リクエストを送りやすくするとか、受け取ったことを気付きやすくするための改修の方が先に着手すると思われます）

今できる対応としては、正直手間がかかって申し訳ないのですが
https://qiita.com/patches
こちらのURLにアクセスいただくと自分が受信・送信した編集リクエストが一覧になっています
間違って送ってしまった際は、ここから一度削除をして再送信……をお願いできればと思っています 🙏

**Reply by daddygongon** [reply]
早速のご返答ありがとうございます．利用者を増やすには，新規顧客を開拓するよりリピータを増やすことが手っ取り早くって，私のような経験をされた方はその機能を使うことを躊躇されると思います．ただ，削除操作があることは気が付いていなかったので，そちらで対応します．新しい機能は慣れると便利なんですが，気が付かないことが多くって...ご教示助かります．

**Reply by daikisuyama** [reply]
@xrxoxcxox 

編集リクエストを定期的に利用しているため、https://qiita.com/patches は非常に便利であると感じました。こちらのページへのリンクはユーザページやトップページなどにありますでしょうか？
もしない場合は、今後の編集リクエスト機能の改善をする際にわかりやすいところに設置していただけるとありがたいです。

**Reply by uhooi** [reply]
横から失礼します 🙏 
このページ初めて知りました…！ めちゃ便利ですねｗ

**Reply by kskwtnk** [reply]
@daddygongon @daikisuyama @uhooi 

ヘッダー内の自分のアイコンをクリックして表れるメニューの中の「編集リクエスト」よりアクセスいただけます！

![](https://user-images.githubusercontent.com/22670303/136188040-66d8c9d3-a094-433f-bb92-5c48b3ee8295.png)

こちらのリンク、一目で何があるかは分かりづらいと思いますので、動線の位置やワーディングを分かりやすくできるようにします 🙏

**Reply by daikisuyama** [reply]
@xrxoxcxox 
教えていただきありがとうございます。
今後は利用させていただきます。
